packages/components/src/components/primitives/Snackbar/Snackbar.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { SnackbarType } from './Snackbar';
+
+export const getStyle = ({
+  isTextRightPosition = false,
```

Comment:
having the position as a boolean sounds weird to me

Reply:
Any suggestions? Somehow I need to control the position where the text is going to be displayed and adjust the flex order based on this "boolean".

Reply:
for a position attribute I will go for an enum, not sure if a bit overkill is this case

Reply:
Not sure if that is what you meant but I've added   
`const isTextRightPosition = textPosition === SnackbarTextPosition.Right;`

Reply:
what I really meant is that we should pass the textPosition to this components instead of the boolean, but that's ok for now